content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -183,7 +183,6 @@ Shutdown the [sleep]({{< github_tree >}}/samples/sleep) and [httpbin]({{< github
 $ kubectl delete telemetry sleep-logging
 $ kubectl delete -f @samples/sleep/sleep.yaml@
 $ kubectl delete -f @samples/httpbin/httpbin.yaml@
-$ kubectl delete -f @samples/open-telemetry/otel.yaml@
```

Comment:
Why is this being deleted? It is needed, since the doc creates in the before you begin section.

Reply:
Added back with the namespace where is was applied.